query-engine/driver-adapters/js/adapter-pg/package.json

```
@@ -1,6 +1,6 @@
 {
-  "name": "@jkomyno/prisma-adapter-pg",
-  "version": "0.2.1",
+  "name": "@prisma/adapter-pg",
+  "version": "0.0.0",
```

Stay with current version (manually increased) until we have CI setup:
```suggestion
  "version": "0.2.1",
```